src/commands/postDeploy/getLinuxDetectorError.ts

```diff
@@ -3,83 +3,97 @@
+export async function getLinuxDetectorError(context: IActionContext, detectorId: string, node: SiteTreeItem, startTime: string, endTime: string, deployEndTime: string): Promise<string | undefined> {
     const detectorUri: string = `${node.id}/detectors/${detectorId}`;
     const requestOptions: requestUtils.Request = await requestUtils.getDefaultAzureRequest(detectorUri, node.root);
-    const dayAfterDeployResultTime: Date = new Date(deployResultTime.getTime() + (24 * 60 * 60 * 1000));
 
-    // these parameters were specified to retrieve the timestamp from the detector by Detectors team
-    // string "Latest time seen by detector. To be used in VSCode integration."" is added to response
-    // when val: 'vscode' is added
     requestOptions.qs = {
         'api-version': "2015-08-01",
-        fId: "1",
-        btnId: "2",
-        val: "vscode",
-        inpId: "1",
-        startTime: deployResultTime.toISOString(),
-        endTime: dayAfterDeployResultTime.toISOString()
+        startTime,
+        endTime,
+        // query param to return plain text rather than html
+        logFormat: 'plain'
     };
 
     const detectorResponse: string = await requestUtils.sendRequest<string>(requestOptions);
     const responseJson: detectorResponseJSON = <detectorResponseJSON>JSON.parse(detectorResponse);
 
-    const datasets: detectorDataset[] = responseJson.properties.dataset;
-    const vsCodeIntegration: string = 'Latest time seen by detector. To be used in VSCode integration.';
-    const timestampTable: detectorTable | undefined = findTableByRowValue(datasets, vsCodeIntegration);
+    const insightLogTable: detectorTable | undefined = findTableByName(responseJson.properties.dataset, 'insight/logs');
 
-    // if we can't find the timestamp, exit and try again
-    if (!timestampTable) {
+    if (!insightLogTable) {
         return undefined;
     }
 
-    const timestamp: Date = new Date(getValuesByColumnName(context, timestampTable, ColumnName.value));
-    if (validateTimestamp(context, timestamp, deployResultTime)) {
-
-        const criticalErrorTable: detectorTable | undefined = findTableByRowValue(datasets, 'Critical');
-        const failureCountTable: detectorTable | undefined = findTableByColumnName(datasets, ColumnName.failureCount);
-
-        // if the criticalError table or failureCounts aren't defined, we can't display anything to the user
-        if (!criticalErrorTable || !failureCountTable) {
+    const rawApplicationLog: string = getValuesByColumnName(context, insightLogTable, ColumnName.value);
+    const insightDataset: detectorDataset[] = <detectorDataset[]>JSON.parse(rawApplicationLog);
+
+    let insightTable: detectorTable;
+    let detectorTimestamp: string;
+
+    // [1] 2020-04-21T18:23:50 is the format of the timestamp in the insight response, this RegExp will remove the brackets
+    const bracketsAndSpace: RegExp = /\[.*?\]\s/;
```

Comment:
Please modify the above comment to explain why there is a bracket thing in the timestamp. Basically what you explained to me in the GitHub comments